api/views/books.py

```diff
@@ -1,16 +1,7 @@
-from flask import Blueprint, jsonify
-from .schemas import BookSchema
-from .mock_db import db
-
-
-def jsonify_schema_dump(*args, **kwargs):
-    """Wrapper around jsonify that sets the Content-Type of the response to
-    application/vnd.api+json.
-    """
-    response = jsonify(*args, **kwargs)
-    response.mimetype = 'application/vnd.api+json'
-    return response
-
+from flask import Blueprint
+from api.mock_db import db
```

Comment:
How can we abstract away the data source we are using? 
What about the serialization part?
Why would it be beneficial to do it?